docs/pages/monitoring/nuts-go.rst

```diff
@@ -28,10 +28,173 @@ It'll return some text displaying the current status of the various services
 
 .. code-block:: text
 
-Status
-	Registered engines: Status,Logging,Crypto,Registry,Events octopus,ConsentLogicInstance,ConsentStore,Validation,Auth,ConsentBridgeClient
-Logging
-	verbosity:
-Events octopus
-	Nats streaming server: mode: STANDALONE @ 0.0.0.0:4222, ID: nuts, last error: NONE
-	DB: ping: true
+    Status
+        Registered engines: Status,Logging,Crypto,Registry,Events octopus,ConsentLogicInstance,ConsentStore,Validation,Auth,ConsentBridgeClient
+    Logging
+        verbosity:
+    Events octopus
+        Nats streaming server: mode: STANDALONE @ 0.0.0.0:4222, ID: nuts, last error: NONE
+        DB: ping: true
+
+Metrics
+*******
+
+The Nuts service executable has build-in support for **Prometheus**. Prometheus is a time-series database which supports a wide variety of services. It also allows for exporting metrics to different visualization solutions like **Grafana**. See https://prometheus.io/ for more information on how to run Prometheus. The metrics are exposed at ``/metrics``
+
+Configuration
+=============
+
+In order for metrics to be gathered by Prometheus. A ``job`` has to be added to the ``prometheus.yml`` configuration file.
```

Comment:
Can we include a default configuration for prometheus so that there are metrics by default without having to configure external resources (which can be a hassle in e.g. Kubernetes)?

Reply:
you have to configure IP/domain and port in your config. Env variables are not possible so a default config won't work.